nodes/nodes/ecosystems/eco-machine-health-checks.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="machine-health-checks"]
+= Remediating nodes with Machine Health Checks
+include::_attributes/common-attributes.adoc[]
+:context: machine-health-checks
+
+toc::[]
+
```

Comment:
Please add a sentence or a short introductory paragraph to serve as the abstract.

Reply:
Thanks @mjpytlak I've applied your feedback.